index.yaml

```diff
@@ -10,8 +10,46 @@ indexes:
 # automatically uploaded to the admin console when you next deploy
 # your application using appcfg.py.
 
+- kind: BridgeCrew
```

Comment:
We should probably update the version of `index.yaml` in the repo or `.gitignore` it, but I do not think this is the place.

Reply:
Rebasing of `master` (where `index.yaml` was removed from the repo and added to the `.gitignore`) should hopefully resolve this.